src/documents/WebGamesAndSimulations.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Web Games and Simulations
+layout: default
+category: Methods
+---
+
+Interactive games can provide rich learning experiences for many students - but for some, these games are a accessibility and inclusivity challenge. This guide aims at giving some practical techniques to help improve the inclusiveness and accessibility of interactive content whether retrofitting existing content or creating a new game from scratch.
+
+**Scope of this Guide:** This guide deals with interactive games built using standard web technologies such as HTML5, CSS, SVG, and JavaScript. By using established web standards, the content you create has a higher chance of being understood by assistive technologies.
+
+## Orienting the User
+
+Regardless of the modality used for interaction, a logical, predictable structure helps users focus on learning and the experience instead of getting stuck on the interaction. Consistency and predicability is important.
```

Comment:
I think this sentence might flow better – and still say the right thing – if you remove the phrase "and the experience" after "focus on learning"